docs/07-deployments/04-deploying-to-aws.md

```diff
@@ -118,6 +132,27 @@ _Connect to the database with Postico._
 
 ## Deploy your code
 
+:::caution
+
+using an old version of serverpod cli will generate the github action file containing old dart version. You can make the following fixes:
+
+1. in `./github/workflows/deployment-aws.yml` use
+
+```yaml
+      - name: Setup Dart SDK
+        uses: dart-lang/setup-dart@v1.6.5
+        with:
+          sdk: 3.5
+```
+
+1. in `mypod_server/config/production.yaml` change the version of installed dart like example
```

Comment:
Question: This looks wrong, there are no dart versions defined in production.yaml should be some other file right?

Also capitalize the first letter and add a . at the end.